src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,37 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    protected LocalDate deadline;
+
+    Deadline(String description, String deadline) {
```

Comment:
Constructors here are initialized with the `default` access modifier. `default` makes them package-private. You could have made them `public` for accessing across packages, if needed.